server/assets/tests/specs/invoiceItemSpec.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+var chai = require('chai');
+var superagent = require('superagent');
+
+describe('Invoice item tests', function() {
+
+  var URL = 'https://localhost:4443';
+  var invoiceItemObj = {
+    invoiceId: "558317feee627fa68f5aab21",
+    description: 'item on invoice',
+    qty: 5,
+    rate: 10,
+    isFlatFee: true
+  };
+  var currentItemId;
+  var currentInvoiceId = "558317feee627fa68f5aab21";
+  var modifiedItem = {
+    invoiceId: "558039a859496aeaecffeda8",
+    description: 'item on invoice',
+    qty: 100,
+    rate: 10,
+    isFlatFee: true
+  };
+  /*jshint -W030 */
+  describe('CRUD actions', function() {
+
+    describe('Create', function() {
```

Comment:
It's referencing an invoice that already exists in the database with the currentInvoiceId variable